### ISSUE TITLE
NPE thrown in very rare cases where jannovar fails to fully annotate an intergenic variant 

### ISSUE THREAD

**Issue by julesjacobsen** [exomiser/Exomiser#224]
When an INTERGENIC_VARIANT in a TAD is reassigned to be a REGULATORY_REGION_VARIANT and has not been assigned a gene symbol or identifier by Jannovar a ```NullPointerException``` is thrown: 

```
Exception in thread "main" java.lang.NullPointerException
        at org.monarchinitiative.exomiser.core.analysis.util.GeneReassigner.prioritiserScore(GeneReassigner.java:109)
        at org.monarchinitiative.exomiser.core.analysis.util.GeneReassigner.assignVariantToGeneWithHighestPhenotypeScore(GeneReassigner.java:82)
        at org.monarchinitiative.exomiser.core.analysis.util.GeneReassigner.reassignRegulatoryRegionVariantToMostPhenotypicallySimilarGeneInTad(GeneReassigner.java:74)
```

Fix this so that the variant is either re-assigned to gene in the TAD with the highest prioritiser score, or is left where it is and ignored in the output.

**Comment by damiansm** [comment]
Tested and looks good to merge back into master and for a hot fix for release 8.0.0

**Comment by julesjacobsen** [comment]
New release created: https://github.com/exomiser/Exomiser/tree/8.0.1
